packages/cursorless-vscode/src/ide/vscode/VscodeFileSystem.ts

```diff
@@ -1,26 +1,87 @@
-import { Disposable, FileSystem, PathChangeListener } from "@cursorless/common";
-import { RelativePattern, workspace } from "vscode";
-import * as path from "path";
+import {
+  Disposable,
+  FileSystem,
+  PathChangeListener,
+  RunMode,
+} from "@cursorless/common";
+import { isAbsolute } from "path";
```

Comment:
I removed the import of `path` on purpose.  The goal is to remove dependencies on node modules so the extension will run in a web host with minimal effort. Do we really need isAbsolute vs startsWith("/")?

Reply:
Will that work on windows?

Reply:
I think I'd prefer to use a polyfill. See eg https://github.com/evanw/esbuild/issues/85#issuecomment-624346338. Is that going to cause a problem for your compiler?